src/libs/tagging.c

```diff
@@ -506,6 +567,7 @@ static gboolean _find_tag_iter_tagid(GtkTreeModel *model, GtkTreeIter *iter, gin
   return FALSE;
 }
 
+#if 0
```

Comment:
Either the code should be removed or some comments added to why it is commented out.

Reply:
I'm in favor of removing, it is in Git anyway.

Reply:
>  it is in Git anyway.

True. I'll remove it.